src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let newState = [];
+  const finalState = [];
+
+  newState = { ...state };
```

Comment:
The initialization of newState as an empty array seems unnecesary since you're reassigning it as an object straight away, maybe just discard line 10 and initialize it like in line 13